hack/mosquitto-passwd.txt

```diff
@@ -0,0 +1 @@
+maestro:$7$101$RI3O9mptIyQG7gQA$NTzK5AXPUDQcaOaqa/Y7yzvofjUt/t47N8TIdEVyjWZu/dKbKxbhjiNdZWvM2t6kLOljC644cItZpX8j0NcDGQ==
```

Comment:
there is the /secrets directory.   And .conf would be by deployment, so this belongs in /test if for our integration tests.

Reply:
I presume, too, that the secret is bogus for a local test deployment. 

I know TAP and gitleaks don't like secrets (even test ones) in the repo.  

if this is only for testing purposes, can the secret by created dynamically to avoid failing a security scan?

Reply:
yes, the secret is only for local testing.

A dynamically generated password is an option, but it needs the 'mosquitto_passwd' binary to generate the `password.txt`. Is it acceptable to include it?

Reply:
update: use the `mosquitto_passwd` binary from docker container to generate the dynamic password for mqtt broker.